dev-python/PyQtWebEngine/PyQtWebEngine-5.12.1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+inherit python-r1 qmake-utils
+
+DESCRIPTION="Python bindings for Qt WebEngine framework"
+HOMEPAGE="https://www.riverbankcomputing.com/software/pyqtwebengine/intro"
+
+MY_P=${PN}_gpl-${PV/_pre/.dev}
+SRC_URI="https://www.riverbankcomputing.com/static/Downloads/${PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug"
+
+RDEPEND="${PYTHON_DEPS}
+	>=dev-python/PyQt5-5.12.1[${PYTHON_USEDEP},widgets]
+	>=dev-python/PyQt5-sip-4.19.14:=[${PYTHON_USEDEP}]
+	>=dev-qt/qtwebengine-5.12[widgets]"
+DEPEND="${RDEPEND}
+	>=dev-python/sip-4.19.14[${PYTHON_USEDEP}]"
+BDEPEND=""
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	default
+	python_copy_sources
```

Comment:
`configuration()` uses an out-of-source build, so why are you copying the sources here?